### ISSUE TITLE
‘l’, ‘O’, or ‘I’ names not checked in function parameters

### ISSUE THREAD

**Issue by atzannes** [PyCQA/pycodestyle#863]
The following code does not raise any warnings:
```
def f(l):
    return l
```
It seems that E741, E742, and E743 do not check function parameter names for bad names, but [PEP8 (names to avoid)](https://www.python.org/dev/peps/pep-0008/#names-to-avoid) does not seem to make a special exception for those.

**Comment by asottile** [comment]
I believe #700 has the rationale for this choice

**Comment by atzannes** [comment]
Actually, it looks like there is some debate about whether to flag function parameters vs arguments at call-sites, but most seem to agree that we should flag function parameters (at function declarations) but not function arguments (at function call-sites). PR #853 seems to implement that logic, but has not been released yet, so I'll wait for the next release of pycodestyle.

@asottile thanks for the helpful pointer.

I'm going to close this issue as it appears to have been resolved by PR #853

**Comment by asottile** [comment]
ooh neat!  cheers :tada:
